charts/kafka-connect/README.md

```diff
@@ -0,0 +1,97 @@
+| ---------------------------------------------------------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------ |
+| `global.imageRegistry`                                     | Global Docker image registry                                                  | `""`                                                               |
+| `global.imagePullSecrets`                                  | Global Docker registry secret names as an array                               | `[]`                                                               |
+| `global.storageClass`                                      | Global StorageClass for Persistent Volume(s)                                  | `""`                                                               |
+| `image.registry`                                           | Kafka connect image registry                                                  | `docker.io`                                                        |
+| `image.repository`                                         | Kafka connect image repository                                                | `confluentinc/cp-kafka-connect`                                    |
+| `image.tag`                                                | Kafka connect image tag (immutable tags are recommended)                      | `6.1.0`                                                            |
+| `image.pullPolicy`                                         | image pull policy                                                             | `IfNotPresent`                                                     |
+| `image.pullSecrets`                                        | Kafka connect image pull secrets                                              | `[]`                                                               |
+| `nameOverride`                                             | String to partially override common.names.fullname                            | `""`                                                               |
+| `fullnameOverride`                                         | String to fully override common.names.fullname                                | `""`                                                               |
+| `commonLabels`                                             | Labels to add to all deployed objects (sub-charts are not considered)         | `{}`                                                               |
+| `commonAnnotations`                                        | Annotations to add to all deployed objects                                    | `{}`                                                               |
+| `replicaCount`                                             | to decide the number of kafka connect pod replicas                            | `1`                                                                |
```

Comment:
Description should start with capital letter

Reply:
done